x-pack/filebeat/module/aws/vpcflow/ingest/pipeline.yml

```diff
@@ -0,0 +1,82 @@
+description: Pipeline for AWS VPC Flow Logs
+
+processors:
+  # Convert Unix epoch to timestamp
+  - date:
+      field: "aws.vpcflow.end"
```

Comment:
I think you could drop the original `aws.vpcflow.start/end` fields since they are now recorded in other places.

Reply:
done 😄